File: 1267 핸드폰 요금.py
n = int(input())
call = list(map(int, input().split())) # list로 받아서

y = 0
m = 0

for i in call: # for문 돌릴 때 list에서 돌림
    y += (i//30 + 1) * 10 # 돌리는 값은 i
    m += (i//60 + 1) * 15

if y > m:
    print("M %d" % m)
elif y < m:
    print("Y %d" % y)
else:
    print("Y M %d" % y)

File: 1267 핸드폰 요금.c
#define _CRT_SECURE_NO_WARNINGS
#include<stdio.h>

int main() {

    int n, call;
    int y = 0, m = 0;
    
    scanf("%d", &n);

    for (int i = 0; i < n; i++) {
        scanf("%d", &call);

        y += (call / 30 + 1) * 10;
        m += (call / 60 + 1) * 15;
    }

    if (y > m) {
        printf("M %d", m);
    }
    else if (y < m) {
        printf("Y %d", y);
    }
    else if(y == m) {
        printf("Y M %d", y);
    }

    return 0;
}
